Utilities/DataFlow/CMakeLists.txt

```diff
@@ -94,5 +97,8 @@ O2_GENERATE_TESTS(
 
 O2_GENERATE_MAN(NAME TimeframeReaderDevice)
 O2_GENERATE_MAN(NAME TimeframeWriterDevice)
+O2_GENERATE_MAN(NAME SubframeBuilderDevice)
 
 target_compile_options(test_TimeframeParser PUBLIC -O0 -g)
+target_compile_options(test_PayloadMerger01 PUBLIC -O0 -g)
+target_compile_options(test_SubframeUtils01 PUBLIC -O0 -g)
```

Comment:
I'm also confused about hardwired compiler options. The flags are just appended and thus override any other optimization settings. The CI never has the chance to build the tests with some other setting. Even if this is ok for the tests, it's a bad example to have this in the cmake setup. It somehow undermines the idea that we globally take care of the cmake settings and dependencies.